packages/react/testing-library/api-extractor.json

```diff
@@ -4,12 +4,21 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
   "extends": "../../../api-extractor.json",
-  "mainEntryPointFilePath": "<projectFolder>/types/index.d.ts",
+  "mainEntryPointFilePath": "<projectFolder>/types/entry.d.ts",
   "compiler": {
     "overrideTsconfig": {
       "compilerOptions": {
         "skipLibCheck": true,
       },
     },
   },
+  "bundledPackages": [
+    "@testing-library/dom",
+    "@lynx-js/test-environment",
+  ],
+  "dtsRollup": {
+    "enabled": true,
```

Comment:
If we are using `dtsRollup` directly from `api-extractor`, I think we need to run `api-extractor` before/after `rslib build`.

Rslib also supports bundle types with api-extractor. Can we directly use that?